docs/Documentation/UserGuide/6-System Tools/7-Monitor and Log Tools.md

```diff
@@ -357,3 +357,115 @@ Here are the file size statistics:
 |Timeseries Name| root.stats.file\_size.SCHEMA |
 |Reset After Restarting System| No |
 |Example| select SCHEMA from root.stats.file\_size.SCHEMA|
+
+## Performance Monitor
+
+### Introduction
+
+In order to grasp the performance of iotdb, we add this module to count the time-consuming of each operation. This module can statistic the avg time-consuming of each operation and the proportion of each operation fall into a time range. The output is in log_measure.log file. A output example is in below.  
+
+<img style="width:100%; max-width:800px; max-height:600px; margin-left:auto; margin-right:auto; display:block;" src="https://user-images.githubusercontent.com/13203019/60937461-14296f80-a303-11e9-9602-a7bed624bfb3.png">
+ 
+### Configuration parameter
+
+location：conf/iotdb-engine.properties
+
+<center>**Table -parameter and description**
+
+|Parameter|Default Value|Description|
+|:---|:---|:---|
+|enable\_performance\_stat|false|Is stat performance of sub-module enable.|
+|performance\_stat\_display\_interval|60000|The interval of display statistic result in ms.|
+|performance_stat_memory_in_kb|20|The memory used for performance_stat in kb.|
+</center>
+ 
+### JMX MBean
+
+Connect to jconsole with port 31999，and choose ‘MBean’in menu bar. Expand the sidebar and choose 'org.apache.iotdb.db.cost.statistic'. You can Find：
+ 
+<img style="width:100%; max-width:600px; max-height:200px; margin-left:auto; margin-right:auto; display:block;" src="https://user-images.githubusercontent.com/13203019/60937484-30c5a780-a303-11e9-8e92-04c413df2088.png">
+
+**Attribute**
+
+1. EnableStat：Whether the statistics are enable or not, if it is true, the module records the time-consuming of each operation and prints the results; It can not be set dynamically but changed by function in below.
```

Comment:
"enable" -> "enabled"
"It can not be set dynamically but changed by function in below." -> "It is non-editable but can be changed by the function below."